### ISSUE TITLE
Skip links failing 1.4.13 content on hover/focus

### ISSUE THREAD

**Issue by alastc** [w3c/wcag#1650]
(Chair hat off, consultant with client hat on...)

Where you have an invisible skip link that becomes visible as you are tabbing through, and it covers other content in some page variations, that seems to fail the letter of [1.4.13](https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus). 

For example:
<img src="https://user-images.githubusercontent.com/216630/108498971-86b34d00-72a5-11eb-8cd1-b69ee544379c.png" alt="Screenshot from Gov.uk showing the skip link over the logo.">

(From [gov.uk](https://www.gov.uk/), not our client, but another example.)

It covers the logo, but this seems trivial to the point of being a false-positive.

I don't remember this coming up in the scenarios we discussed during 2.1 development, which were more focused on the pointer-hover scenarios. 

Does anyone disagree with the interpretation? Is it worth some sort of exception/errata?

**Comment by innoticFR** [comment]
Maybe the note [in the understanding](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html) was supposed to cover that aspect?

>This SC covers content that appears in addition to the triggering component itself. Therefore, a non-visible component, like a Skip to Main link, that is made visible on keyboard focus (with no additional content beyond the trigger becoming visible) is not covered by the SC.

**Comment by patrickhlauke** [comment]
Yes, the note in understanding exempts this scenario...but of course one can now argue over "that's only non-normative, but this case does clearly violate the normative wording..."

or we can thin-slice "when the normative wording says 'additional content' ... what is it additional to? the page itself (which would cover the skip link)? or additional to the trigger element? (which then would fall more in line with the understanding note)"

**Comment by awkawk** [comment]
This was definitely not intended to be covered by the SC. I don't think that the "additional" is slicing too thin - the note addresses this.

**Comment by mraccess77** [comment]
I'd say the link itself is not additional content and thus not a violation.

**Comment by alastc** [comment]
Ah, thanks everyone. 

Note to self: You don't remember the understanding docs that well!

**Comment by bruce-usab** [comment]
I do see that @alastc wants to close this.  But as I read 1.4.13 <q>additional content</q> in this example is <q>Skip to main content</q>.

With this example, what is the mechanism to dismiss the additional content without moving pointer hover or keyboard focus?

**Comment by awkawk** [comment]
@bruce-usab Does the understanding note not clarify this for you? If we meant any content then we would have just written "Where receiving and then removing pointer hover or keyboard focus triggers <del>additional</del> content to become visible and then hidden, the following are true:" - we added the "additional" to indicate that it was apart from the target and added the understanding note to make it more clear.

**Comment by bruce-usab** [comment]
Thanks @awkawk.  No, the Understanding note does not clarify this for me because I don't think it closely addresses the question raised by OP.  What I think does address this question for someone taking the SC literally is a peak at [benefits](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html#benefits).

With the benefit of hindsight, I am now of the opinion that the SC text is missing a parenthetical:

Where receiving and then removing pointer hover or keyboard focus triggers additional content <ins>(other than the target)</ins> to become visible and then hidden.

Somewhat ironically, I am basically arguing the opposite case with @patrickhlauke in https://github.com/w3c/wcag/issues/1647

**Comment by patrickhlauke** [comment]
> Somewhat ironically, I am basically arguing the opposite case with @patrickhlauke in #1647

because here it's easy to define what is perhaps missing in the normative wording (basically, the thin-slicing i jokingly alluded to earlier in this thread), whereas on #1647 we'd end up writing a small essay in a parenthetical and probably still not manage to more clearly define what is effectively a giant handwave in the understanding ;)
